### DISCUSSION TITLE
Послушник / Маг Огня

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#77] [General]
```b_startotherroutine(agon,"StillAlive");```

можно менять расписание Агону в Диалоге  с Пирокаром 
func void dia_pyrokar_oath_info()

на тот случай если герой вступить в Маги Огня не в первой главе.

***

**Comment by Kor-Angar** [comment]
вот тут лучше явно прописать мага огня и послушника

```
		if((hero.guild == GIL_MIL) || (hero.guild == GIL_PAL))
		{
			Wld_InsertItem(itam_dex_01,"FP_ROAM_XARDAS_SECRET_26");
		}
		else if((hero.guild == GIL_SLD) || (hero.guild == GIL_DJG))
		{
			Wld_InsertItem(itam_hp_01,"FP_ROAM_XARDAS_SECRET_26");
		}
		else
		{
			Wld_InsertItem(itam_hp_mana_01,"FP_ROAM_XARDAS_SECRET_26");
		};
```

а амулет жизни сделать прочим (наёмник , нейтрал,  онд)

**Comment by Kor-Angar** [comment]
это потому что явная точка в таверне не указана? :-)

![20210302103937_1](https://user-images.githubusercontent.com/49514764/109614471-bc78f100-7b43-11eb-869c-53f5dd0b7cc1.jpg)

а я то удивлялся куда он пропадал :-) а он оказывается просто не мог понять где ему тусить :-)

**Comment by dosinabox** [comment]
> на тот случай если герой вступить в Маги Огня не в первой главе.

В классике возможна такая ситуация?

> это потому что явная точка в таверне не указана? :-)

Указана несуществующая точка, нужно TAVERNE.

**Reply by Kor-Angar** [reply]
> В классике возможна такая ситуация?

да. на диалог с Хагеным нет ограничения на гильдию. поэтому и нейтрал может прейти во вторую главу и послушник

**Reply by Kor-Angar** [reply]
> А где оно меняется сейчас?

в b_enter_newworld_kapitel_2

**Comment by dosinabox** [comment]
> можно менять расписание Агону в Диалоге с Пирокаром

А где оно меняется сейчас?

**Comment by Kor-Angar** [comment]
а это что такое?:-)

```D
	KDF_AUFNAHME = LOG_SUCCESS;
	SLD_AUFNAHME = LOG_OBSOLETE;
	MIL_AUFNAHME = LOG_OBSOLETE;
```

**Comment by Kor-Angar** [comment]
и на этот диалог снять проверку на главу, что бы Маг может высказать пожелание, если стал магом во второй главе.

```D
instance DIA_PYROKAR_WUNSCH(C_INFO)
{
	npc = kdf_500_pyrokar;
	nr = 2;
	condition = dia_pyrokar_wunsch_condition;
	information = dia_pyrokar_wunsch_info;
	permanent = FALSE;
	description = "Я хотел бы высказать пожелание...";
};


func int dia_pyrokar_wunsch_condition()
{
	if((other.guild == GIL_KDF) && (KAPITEL < 2))
	{
		return TRUE;
	};
};
```

**Comment by Kor-Angar** [comment]
в **b_enter_newworld_kapitel_3**

добавить проверку на послушника.

**Comment by Kor-Angar** [comment]
ты напиши, если что то планируешь править. я это сразу проверю. у меня послушник как раз собрался в маги вступать во второй главе.

**Comment by Kor-Angar** [comment]
```D
instance DIA_MILTENOW_TEACHCIRCLE2(C_INFO)
{
	npc = pc_mage_ow;
	nr = 91;
	condition = dia_miltenow_teachcircle2_condition;
	information = dia_miltenow_teachcircle2_info;
	permanent = TRUE;
	description = "Обучи меня второму кругу магии!";
};


func int dia_miltenow_teachcircle2_condition()
{
	if((other.guild == GIL_KDF) 
	&& Npc_KnowsInfo(other,dia_miltenow_lehren) 
	&& (Npc_GetTalentSkill(other,NPC_TALENT_MAGE) < 2))
	{
		return TRUE;
	};
};
```
у мильтена или сделать как у тебя в неоф. обновление, что бы он учил и первому и второму кругу или поменять условие на (Npc_GetTalentSkill(other,NPC_TALENT_MAGE) == 1)

**Comment by dosinabox** [comment]
> а это что такое?:-)

Вступление в маги - успешно, вступление в наемники или ополчение - отменено.

> ты напиши, если что то планируешь править. я это сразу проверю. у меня послушник как раз собрался в маги вступать во второй главе.

Пока не планирую. На днях будет релиз 26 версии обновления.

Не проще ли сделать проверку на гильдию у Хагена?
Нужно ли выгонять Агона в обновлении НВ? В НВ его распорядок дня у таверны не используется.

**Comment by Kor-Angar** [comment]
> Вступление в маги - успешно, вступление в наемники или ополчение - отменено.

отмену наёмника и милицию надо делать наверное в диалоге у Педро

> Не проще ли сделать проверку на гильдию у Хагена?

неа.. изначально в самом начале  тобой этот момент прорабатывался и брались правки из мода немецкого. послушник даже руны убрали во второй главе ;-)

**Comment by Kor-Angar** [comment]
> Нужно ли выгонять Агона в обновлении НВ? В НВ его распорядок дня у таверны не используется.

старая история:-) _я предлагаю один расписать что должно происходит после испытания огнем и высказывания пожелания Пирокару._

https://github.com/dosinabox/g2nr_unofficial_update/issues/77

**Comment by Kor-Angar** [comment]
dia_pyrokar_wunsch_condition - недоступна во второй главе

**Reply by Kor-Angar** [reply]
```D
func int dia_pyrokar_wunsch_condition()
{
	if (other.guild == GIL_KDF) 
    && (KAPITEL < 3)
	{
		return TRUE;
	};
};
```

**Comment by Kor-Angar** [comment]
Магу огня после принятие в круг огня не списывают штрафы

![20220810074028_1](https://user-images.githubusercontent.com/49514764/183834183-3a3107be-e7d1-415a-84eb-ebfd95ecd2d0.jpg)
![20220810073330_1](https://user-images.githubusercontent.com/49514764/183834248-57b0b8f4-f6f9-4182-a258-552f90ad1203.jpg)
![20220810073341_1](https://user-images.githubusercontent.com/49514764/183834274-f6f30eac-7196-4652-bb3b-ccf28044bee3.jpg)

**Comment by Kor-Angar** [comment]
Serpentes квест с акциями не хочет давать до пропажи педро

**Reply by Kor-Angar** [reply]
```D
func int DIA_Serpentes_MinenAnteile_Condition ()
{

    if (hero.guild == GIL_KDF)
    && (Kapitel >= 3)
    && Npc_KnowsInfo(hero,DIA_Pyrokar_BACKFROMOW)
    {
        return TRUE;
    };
    
	if (Pedro_Traitor == TRUE)
	&& ((hero.guild == GIL_SLD) || (hero.guild == GIL_DJG))
	&& (Kapitel >= 3)
    {
        return TRUE;
    };
};
```

**Comment by Kor-Angar** [comment]
в продажи был огненный язык. хватило только на  руну "большой огненная буря" . пришлось выучить навык дергания языка что бы скрафтить руну "огенный дождь".

 нужно учесть это или добавлять язык в продажу или вставлять доп огненных ящеров.

**Comment by Kor-Angar** [comment]
клык тролля. тоже самое. надо вставить тролля или добавить клык в продажу что бы иметь возможность сделать руну. при этом в продаже был рог мракориса который для крафта не используется.

**Reply by Kor-Angar** [reply]
рог мракориса у кости
![20220828171916_1](https://user-images.githubusercontent.com/49514764/187086122-2fb0f01c-9b57-4f99-9cb9-f3c0e3434078.jpg)

**Comment by Kor-Angar** [comment]
сейвы за мага огня 2022 [save games
](https://disk.yandex.ru/d/x1PqAPSaqMMYOQ)

**Comment by Kor-Angar** [comment]
Есть проблема при игре магов. когда одержимые атакуют героя-мага и герой маг отбивается магией, то в какой то момент регистрируется преступление. незнаю актуальна это ситуация для НВ или только для классик. у меня так происходит в классик почти со всеми одержимыми.

**Reply by Kor-Angar** [reply]
![20230301013130_1](https://user-images.githubusercontent.com/49514764/222217917-05ea8795-9c9b-44b7-aadf-12d994d85b37.jpg)
![20230228022132_1](https://user-images.githubusercontent.com/49514764/222217989-fb849904-3ad5-4215-bc95-72a893052dcc.jpg)

**Reply by Kor-Angar** [reply]
фернандо с таймкодом https://www.youtube.com/live/QFy-HIAsV0I?feature=share&t=2804

**Reply by Kor-Angar** [reply]
Бромор с таймкодом https://www.youtube.com/live/QFy-HIAsV0I?feature=share&t=12699

**Reply by Kor-Angar** [reply]
![20230302031320_1](https://user-images.githubusercontent.com/49514764/222433199-6d276ec3-273b-44b2-81ec-f7ea7cf6cc55.jpg)

**Reply by Kor-Angar** [reply]
за Энгрома еще и паладины вписались :-)
